pkg/logger/logger.go

```diff
@@ -32,21 +33,51 @@ var logLevelOrder = map[LogLevel]int{
 type Logger struct {
 	LogLevel LogLevel
 	File     string
+	charm    *log.Logger
 }
 
-func NewLogger(logLevel LogLevel, file string) (*Logger, error) {
+func InitializeLogger(logLevel LogLevel, file string) (*Logger, error) {
```

Comment:
The `NewX` naming convention is the standard way to create and return a new instance of a type.

Reply:
I guess i'm confused now between this and the functions above that were called `GetX` but should also have been `NewX` and these here, without investing too much mental effort.